libdevcore/picosha2.h

```diff
@@ -115,9 +115,8 @@ inline uint32_t ssig1(uint32_t x)
 }
 
 template<typename RaIter1, typename RaIter2>
-void hash256_block(RaIter1 message_digest, RaIter2 first, RaIter2 last)
+void hash256_block(RaIter1 message_digest, RaIter2 first, [[maybe_unused]] RaIter2 last)
```

Comment:
Couldn't you just write `void hash256_block(RaIter1 message_digest, RaIter2 first, RaIter2)`?

Reply:
Haha, as a matter of fact, it seems a bit dangerous that this one is unused here - as the "FIXME" comment below confirms :-D.

Reply:
oh true